packages/cysync-core/src/pages/MainApp/Settings/Tabs/About.tsx

```diff
@@ -32,15 +32,15 @@ export const About: React.FC = () => {
         title={{ text: item.termsOfUse.title }}
         description={{ text: item.termsOfUse.description }}
       >
-        <Link to="https://www.cypherock.com/" target="_blank">
+        <Link to="https://www.cypherock.com/terms" target="_blank">
```

Comment:
Link tags needs to be replaced with anchor tags. Refer: `packages\cysync-core\src\dialogs\HistoryDialog.tsx : line 167`

Reply:
fixed